Views/backend/connect/view/config/import/form.js

```diff
@@ -299,7 +310,61 @@ Ext.define('Shopware.apps.Connect.view.config.import.Form', {
             ]
         });
 
-        return [ containerTop, containerMiddle, containerBottom ];
+        elements.push(productContainer);
+        elements.push(imageContainer);
+        elements.push(orderContainer);
+        return elements;
```

Comment:
Not sure right now what Ext.create() returns but maybe this can be simplified with:
```

elements.push(productContainer, imageContainer, orderContainer);
```